solidity/random-beacon/hardhat.config.ts

```diff
@@ -0,0 +1,54 @@
+import { HardhatUserConfig } from "hardhat/config"
+
+import "@keep-network/hardhat-local-networks-config"
+import "hardhat-deploy"
+import "@tenderly/hardhat-tenderly"
+
```

Comment:
I've seen we've got `hardhat-gas-reporter` dependency added to `package.json`, we need to import the plugin here as well.
```suggestion
import "hardhat-gas-reporter"

```

Reply:
0a22a2b